app/screens/Dashboard.js

```diff
@@ -1,24 +1,43 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { SafeAreaView, StyleSheet, ScrollView, View } from 'react-native'
 import { colors } from 'app/config'
+import { fetchData } from 'app/thunks'
+import { useSelector, useDispatch } from 'react-redux'
+import { selectData, selectDataLoading } from 'app/store/reducers/rackings'
 import SystemCell from '../components/common/SystemCell'
-
 const Dashboard = ({ navigation }) => {
-  const imageUrl =
-    'https://images.unsplash.com/photo-1531925470851-1b5896b67dcd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=934&q=80'
+  const dispatch = useDispatch()
+  useEffect(() => {
+    dispatch(fetchData())
+  }, [])
+  const rackingData = useSelector((state) => state.rackings.data)
+  const { systems } = rackingData
   return (
     <SafeAreaView style={styles.container}>
       <ScrollView
         contentInsetAdjustmentBehavior="automatic"
         style={styles.scrollView}
       >
         <View style={styles.container}>
-          <SystemCell
-            imageUrl={imageUrl}
-            title="Pallet Racking"
-            iconName="hammer-outline"
-            onPress={() => navigation.navigate('Info')}
-          />
+          {systems &&
+            systems.map((system, index) => {
+              const { name, icon, header_image } = system.fields
+              return (
+                <SystemCell
+                  key={index}
+                  title={name}
+                  iconName={icon.name}
+                  imageUrl={header_image}
+                  onPress={() =>
+                    navigation.navigate('Info', {
+                      systemId: system.id,
+                      rackingChildren: system.children.rackings,
+                      title: name,
+                    })
+                  }
+                />
+              )
+            })}
```

Comment:
You could use a FlatList here to render these items. https://reactnative.dev/docs/flatlist
It has a lot of handy props such as setting a component to render if no data is available and adding pull to refresh.